assemble/pom.xml

```diff
@@ -171,6 +171,9 @@
             <excludes>
               <exclude>bin-LICENSE</exclude>
               <exclude>bin-NOTICE</exclude>
+              <exclude>nbproject/**</exclude>
+              <exclude>nb-configuration.xml</exclude>
+              <exclude>nbactions.xml</exclude>
```

Comment:
QQ about the xml files: are these created by netbeans when you import the project?